sdk/eventgrid/eventgrid/recordings/browsers/eventgridpublisherclient_send_cloudevent_schema/recording_sends_a_single_event.json

```diff
@@ -10,33 +10,33 @@
         "aeg-sas-key": "api_key",
         "Connection": "keep-alive",
         "Content-Length": "241",
-        "Content-Type": "application/cloudevents-batch\u002Bjson; charset=UTF-8",
+        "Content-Type": "application/cloudevents-batch\u002Bjson; charset=utf-8",
         "Referer": "http://localhost:9876/",
         "sec-ch-ua": "",
         "sec-ch-ua-mobile": "?0",
         "sec-ch-ua-platform": "",
         "Sec-Fetch-Dest": "empty",
         "Sec-Fetch-Mode": "cors",
         "Sec-Fetch-Site": "same-site",
-        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/93.0.4577.0 Safari/537.36",
-        "x-ms-client-request-id": "313d25c0-03aa-4f5a-ad67-e4775337306e",
-        "x-ms-useragent": "azsdk-js-eventgrid/4.7.1 core-rest-pipeline/1.5.1 OS/Linuxx86_64"
+        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/101.0.4950.0 Safari/537.36",
+        "x-ms-client-request-id": "49edd9ad-63f4-4643-9a6d-9740b1d5786a",
+        "x-ms-useragent": "azsdk-js-eventgrid/4.10.0-beta.2 core-rest-pipeline/1.8.2 OS/Linuxx86_64"
       },
-      "RequestBody": "[{\u0022id\u0022:\u0022cloudSingleEventId100406\u0022,\u0022source\u0022:\u0022/earth/unitedstates/washington/kirkland/finnhill\u0022,\u0022data\u0022:{\u0022hello\u0022:\u0022world\u0022},\u0022type\u0022:\u0022Azure.Sdk.TestEvent1\u0022,\u0022time\u0022:\u00222022-02-09T01:46:42.000Z\u0022,\u0022specversion\u0022:\u00221.0\u0022,\u0022datacontenttype\u0022:\u0022application/json\u0022}]",
+      "RequestBody": "[{\u0022id\u0022:\u0022cloudSingleEventId100922\u0022,\u0022source\u0022:\u0022/earth/unitedstates/washington/kirkland/finnhill\u0022,\u0022data\u0022:{\u0022hello\u0022:\u0022world\u0022},\u0022type\u0022:\u0022Azure.Sdk.TestEvent1\u0022,\u0022time\u0022:\u00222022-05-11T00:24:51.000Z\u0022,\u0022specversion\u0022:\u00221.0\u0022,\u0022datacontenttype\u0022:\u0022application/json\u0022}]",
```

Comment:
Doesn't look like this recording needs to be changed?
Can only commit whichever recordings need the updates? To keep the changes minimal.